e2e/benchmarks/local-benchmark/util.js

```diff
@@ -175,3 +175,60 @@ function expectArraysClose(actual, expected, epsilon, key) {
     return tf.test_util.expectArraysClose(actual, expected, epsilon);
   }
 }
+
+async function compareJsons(jsonObject1, backend1, jsonObject2, backend2) {
+  const keys = Object.keys(jsonObject1);
+  var errorCount = 0;
+  for (let i = 0; i < keys.length; i++) {
+    const key = keys[i];
+    let match = false;
+    try {
+      expectObjectsClose(jsonObject1[key], jsonObject2[key]);
+      match = true;
+    } catch (e) {
+      match = false;
+      const newKey = key.replace(/\//g, '-');
+      download(jsonObject1[key], `${i}_${newKey}_${backend1}.json`);
+      download(jsonObject2[key], `${i}_${newKey}_${backend2}.json`);
+      errorCount++;
+      // Without sleep, some files fail to download.
+      await asyncSleep(200);
+    }
+  }
+  if (errorCount) {
+    console.error('Total mismatch: ' + errorCount);
+  }
+}
+
+async function printTensors(tensorsMap, backend) {
```

Comment:
It's more like to collect tensor data instead of print tensors. May a new name is needed?

And the `backend` is not used anywhere.

Reply:
Done